pkg/cli/cmds/agent.go

```diff
@@ -54,8 +53,6 @@ func NewAgentCommand() cli.Command {
 }
 
 func AgentRun(clx *cli.Context) error {
-	if profile == "" {
-		logrus.Warn("not running in CIS 1.5 mode")
-	}
+	validateCISreqs()
```

Comment:
Teasing out the logic of the validateCISreqs, I now wonder if we will refuse to start a worker node because the etcd user doesn't exist even though that user need not exist on a worker node.

Reply:
I think that bug may already exist, but this change here made me think about it.

Reply:
The answer is: yes it already exists.